src/components/Identity/index.js

```diff
@@ -179,6 +178,10 @@ createIdentity.abbreviation = "ID";
 createIdentity.configValidators = {
   idSyncsEnabled: {
     defaultValue: true
+  },
+  idSyncContainerId: {
+    validate: nonNegativeInteger,
```

Comment:
We need to start making our validators more composable. I wrote a JIRA issue for this a while back: https://jira.corp.adobe.com/browse/CORE-32266 In this case, `nonNegativeInteger` would be more flexible if it didn't try to do multiple things. So instead of:

`validate: nonNegativeInteger`

you would do something like:

`validate: number.integer().min(0)`

and later on some other component could do something like:

`validate: number.min(-15).max(250)` or whatever it needed.

If you wanted to do that work now or in a new PR, that would be excellent. Otherwise, add a comment to https://jira.corp.adobe.com/browse/CORE-32266 describing that we need to rework `nonNegativeInteger` later.

Reply:
That would be interesting, but there are other things I need to work on right now.

Regarding IdentityQuery, I did a dangerous merge conflict resolution through the pull request page and messed it up. I won't do that again since it bypasses the commit and push tests.